Issue with json-c and Python 3.6 builds

Recent CircleCI workflows have failed when building CMOR for Python 3.6.  The builds are giving the following errors.

Linux:
```
checking for json_object_new_object in -ljson-c... no
configure: error: Could not get a working json-c
```
OSX:
```
checking for Fortran libraries of ... 
checking for json_object_new_object in -ljson-c... no
configure: error: Could not get a working json-c
```

I will investigate why the json-c library is not working correctly with the Python 3.6 builds.

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -202,7 +202,7 @@ workflows:
               matrix:
                  parameters:
                     os: [ linux, macos ]
-                    python_version: [ "3.6", "3.7", "3.8", "3.9" ]
+                    python_version: [ "3.7", "3.8", "3.9" ]
               name: build-<< matrix.os >>-<< matrix.python_version >>
               requires:
                  - setup-<< matrix.os >>

